packages/combo-box/src/vaadin-combo-box-item.js

```diff
@@ -140,6 +140,7 @@ export class ComboBoxItem extends ThemableMixin(DirMixin(PolymerElement)) {
   }
 
   /** @private */
+  // eslint-disable-next-line max-params
```

Comment:
nit: we could remove `_selected` and `_focused` params instead (that won't affect the logic).

Reply:
Done.